### ISSUE TITLE
Disable Index/Knuckles Radio (and use Wired-only)?

### ISSUE THREAD

**Issue by tineras** [ValveSoftware/openvr#1700]
I have a use case that requires that the controllers be used in a wired-only environment. Curious if anyone knows if there is a custom firmware (or other method) that allows this. I'm also not certain that positional tracking works in a wired-only environment. I know that the buttons and analog sticks do. And I know that this is possible with the OG wands.

- So is there custom firmware for the Knuckles controllers that will allow me to disable the radios?
- Is there a way to get positional tracking working using a wired-only configuration?

I know this is possible for the Vive Pro and the original Wands because there is a "Secure" version that is sold directly from HTC that does exactly that. This also includes being able to disable all radios for the headset and base stations, but that is a solved issue.

Thanks

**Comment by danwillm** [comment]
This would be an issue to raise with Valve specifically, not as a openvr github issue - as this isn't related to openvr.

**Comment by tineras** [comment]
> This would be an issue to raise with Valve specifically, not as a openvr github issue - as this isn't related to openvr.

I agree 100%. I would certainly prefer to deal directly with Valve, but this is where I was told to go when I asked about the issue via the Steam support system. It seems there is no easy way to contact Valve directly. Frustratingly, I actually need this support for a work project. I work for a large, well-known company in the games industry and have struck out when trying to find a POC for Valve using our internal messaging platform.

So this is me grasping at straws to try and get support.

**Comment by danwillm** [comment]
Chances are you won't get a response from Valve here either, seems like Valve have dropped support on openvr github issues - steam support _is_ where you should be looking at contacting them, so it seems strange that they directed you here...

Why exactly do you need to disable the radios?

**Comment by tineras** [comment]
Please don't take this as me being rude, but **why** I need to disable the radios is not relevant. Let's just say that it's a hard requirement in my situation and there are no other options if I want to use these controllers.

**Comment by tineras** [comment]
Here is their response, which obviously was not helpful:

"I apologize, but Steam Support is for technical assistance only. We cannot provide assistance with application development issues.
 For issues or bugs relating to the OpenVR system, we suggest reaching out to the developers directly using the [Valve Software OpenVR Github](https://github.com/ValveSoftware/openvr)."

but again... I came here grasping at straws.
